zebrad/tests/acceptance.rs

```diff
@@ -4,195 +4,292 @@
+
+    Ok(())
 }
 
 #[test]
-fn help_no_args() {
-    let mut runner = RUNNER.clone();
-    let cmd = runner.arg("help").capture_stdout().run();
-    cmd.wait().unwrap().expect_success();
+fn help_no_args() -> Result<()> {
+    zebra_test::init();
+
+    let child = get_child_single_arg("help");
+    let output = child.unwrap().wait_with_output()?;
+    let output = output.assert_success()?;
+
+    output.stdout_contains(r"USAGE:")?;
+
+    Ok(())
 }
 
 #[test]
-fn help_args() {
+fn help_args() -> Result<()> {
+    zebra_test::init();
+
     // Invalid argument
-    let mut runner = RUNNER.clone();
-    let cmd = runner.args(&["help", "argument"]).capture_stdout().run();
-    cmd.wait().unwrap().expect_code(1);
+    let child = get_child_multi_args(&["help", "argument"]);
+    let output = child.unwrap().wait_with_output()?;
+    let output = output.assert_failure()?;
+
+    output.stdout_contains(r"wasn't recognized.")?;
 
     // Invalid flag
-    let mut runner = RUNNER.clone();
-    let cmd = runner.args(&["help", "-f"]).capture_stdout().run();
-    cmd.wait().unwrap().expect_code(1);
+    let child = get_child_multi_args(&["help", "-f"]);
+    let output = child.unwrap().wait_with_output()?;
+    let output = output.assert_failure()?;
+
+    output.stdout_contains(r"option `-f` does not accept an argument")?;
+
+    Ok(())
 }
 
 #[test]
-fn revhex_no_args() {
-    let mut runner = RUNNER.clone();
-    let _cmd = runner
-        .timeout(Duration::from_secs(1))
-        .arg("revhex")
-        .capture_stdout()
-        .run();
+fn revhex_no_args() -> Result<()> {
+    zebra_test::init();
+
+    let child = get_child_single_arg("revhex");
+
+    //Program is waiting for input, we just exit after 1 second
+    std::thread::sleep(Duration::from_secs(1));
+    let mut child_unwrapped = child.unwrap();
+    child_unwrapped.kill()?;
 
-    // Program is waiting for input, we just exit after 1 second
+    let output = child_unwrapped.wait_with_output()?;
+    output.assert_failure()?;
+
+    Ok(())
 }
 
 #[test]
-fn revhex_args() {
-    let mut runner = RUNNER.clone();
-    let mut cmd = runner.args(&["revhex", "33eeff55"]).capture_stdout().run();
-    cmd.stdout().expect_line("55ffee33");
+fn revhex_args() -> Result<()> {
+    zebra_test::init();
+
+    // Valid
+    let child = get_child_multi_args(&["revhex", "33eeff55"]);
+    let output = child.unwrap().wait_with_output()?;
+    let output = output.assert_success()?;
+
+    output.stdout_contains(r"55ffee33")?;
+
+    Ok(())
 }
 
 #[test]
-fn seed_no_args() {
-    let mut runner = RUNNER.clone();
-    let mut cmd = runner
-        .timeout(Duration::from_secs(1))
-        .arg("seed")
-        .capture_stdout()
-        .run();
+fn seed_no_args() -> Result<()> {
+    zebra_test::init();
+
+    let child = get_child_single_arg("seed");
+
+    // Run the program and kill it at 1 second
+    std::thread::sleep(Duration::from_secs(1));
+    let mut child_unwrapped = child.unwrap();
```

Comment:
```suggestion
    let mut child = child.unwrap();
```

TIOLI: It's probably better to just shadow the variable here since we're never going to reuse the original binding to `child`

Reply:
Resolved in previous commit.